resources/js/svir_viewer_d3_tree.js

```diff
@@ -338,7 +337,7 @@
             ))
             {
                 pdTempIds.push(pdData.id);
-                createSpinner(pdData.id, pdData.weight, pdData.name, pdData.field);
+                createSpinner(pdData.id, pdData.weight, pdData.name, pdData.field, pdData.isInverted);
```

Comment:
Nodes now contain an additional field called `isInverted`. A `invert` checkbox is used to populate such field. In this case, pdData is the node.